paper/paper.md

```diff
@@ -0,0 +1,141 @@
+
+# Statement of Need
+
+Comparative research into municipal governance in the USA is often prohibitively difficult due to a broad federal system where states, counties, and cities divide legislative powers differently. This has contributed to the lack of large-scale quantitative studies of municipal government, and impeded necessary research into effective procedural elements of administrative and legislative processes [@Trounstine:2009]. Council Data Project (CDP) enables large-scale quantitative studies by generating standardized municipal governance corpora - including legislative voting records, timestamped transcripts, and full legislative matter attachments (related reports, presentations, amendments, etc.).
+
+## Related Work
+
+Work in extracting and repackaging government data into machine-readable and experiment ready datasets has historically happened in fields with highly structured data, such as meteorology [@Sparks2017] and legal review and monitoring [@courtlistener]. Notably, there has been prior work in extracting and repackaging municipal government data with [Councilmatic](https://github.com/codeforamerica/councilmatic) [@councilmatic]. However, this work largely aimed to make municipal data more accessible to a general public, and didn't add any specific data processing to expand the research capabilities of the produced dataset. Recent advances in natural language processing have made it possible to conduct large-scale transcript-based studies on the effects of gender, ideology, and seniority in Supreme Court oral argument [@jacobi2017] and the effects that information communication technology has on civic participation [@einstein2021].
+
+## CDP Architecture
+
+Council Data Project consists of three primary tools:
+
+1. [cookiecutter-cdp-deployment](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment): A Python [cookiecutter](https://cookiecutter.readthedocs.io/) [@cookiecutter] template to assist users in fully deploying a new CDP instance. A "CDP Instance" is a unique deployment of CDP software and tools. For example, there is an "instance" of CDP for the ["Seattle City Council"](https://councildataproject.org/seattle/#/) and an instance of CDP for the "King County Council". Each instance is comprised of it's own repository, database, file storage bucket, processing pipelines, and web application.
```

Comment:
Should "Seattle City Council" be wrapped in quotes?

Reply:
Same question for King County Council.

Reply:
:shrug: I view it as "this is a unique thing". It's also how i mentally think of instances. We have a "seattle" instance and a "king county" instance. idk. its a nice comparmentalizer.

Reply:
I can get behind that rationale and think it's fine to leave it as is in that case. 🙂